mipengine/controller/cleaner.py

```diff
@@ -128,21 +122,19 @@ def _exec_context_id_cleanup(
         for node_id in node_ids:
             try:
                 node_info = self._get_node_info_by_id(node_id)
+                task_handler = _get_node_task_handler(node_info)
```

Comment:
this try block, as the logging error message implies, is supposed to handle the failing of getting the node's info. As is now, if the task queuing raises an exception the log will still report that the problem was that "Could not get node info for..."